modules/ROOT/pages/vpn-about.adoc

```diff
@@ -16,31 +22,89 @@ To create and maintain a secure connection with your external network, the Anypo
+////
+
+== Anypoint VPN Features
 
-== Supported IPsec Configurations
+=== IPsec Settings
 
-Anypoint VPN supports the IPsec configurations for encrypting data communications between Anypoint Platform and your network:
+Anypoint VPN supports any combination of the following IPsec settings:
 
 * IKE version 1
-* AES 128 and 256-bit encryption
-* SHA and SHA-2 hashing
-* Diffie-Hellman (DH) groups:
-** Phase1 DH groups 2, 14-18, 22, 23, 24
-** Phase2 DH groups 2, 5, 14-18, 22, 23, 24
+* IKE version 2 for route-based VPNs only
+* AES 128 or 256-bit encryption
+* SHA or SHA-2 (256) hashing
+* Diffie-Hellman Phase1 groups 2, 14-18, 22, 23, 24
+* Diffie-Hellman Phase2 groups 2, 5, 14-18, 22, 23, 24
+
+[NOTE]
+Your endpoint must initiate traffic to establish and maintain the connection so the Anypoint VPN endpoint can act as a responder.
+
+=== Anypoint VPN Routing
 
+Anypoint VPN supports dynamic or static routing for VPN connections.
+
+* *Dynamic routing* Your device uses Border Gateway Protocol (BGP) to advertise routes to Anypoint VPN. We recommend using BGP routing if your device supports this protocol.
+* *Static routing* - Requires you to specify the routes (subnets) in your network that are accessible through Anypoint VPN. +
 [NOTE]
-Because the Anypoint VPN endpoint is the responder, be sure to establish and maintain connections by initiating traffic from a remote device.
+A maximum of 95 static routes are permitted per VPC, regardless of the number of VPN connections. We recommend consolidating networks to the fewest number possible.
+
+=== Limitations
+
+Anypoint VPN does not support these features and configurations:
+
+* Network Address Translation (NAT)
+* IPv6
+* IKEv2 with policy-based VPNs
+* A single VPC with both Direct Connect and Anypoint VPN connections
+* Advertising a default route (0.0.0.0/0) over BGP or static routing
+
+[[vpn_requirements]]
+== Requirements
+
+To create a static VPN connection, your VPN endpoint must be able to:
+
+* Establish IKE Security Associations using a Pre-Shared Key (PSK) 
+* Establish IPsec Security Associations in Tunnel mode
+* Utilize any combination of IPsec settings that MuleSoft supports
+* Fragment IP packets before encryption
+* Use one Security Association pair per tunnel
 
-== Anypoint VPN Routing
+=== Fragmentation Before Encryption
 
-The Anypoint VPN supports two types of routing:
+You must fragment packets that are too large to transmit. Your VPN device must be able to fragment packets before encapsulation.
+
+=== Security Association (SA)
+
+Each VPN connection has two separate tunnels. Anypoint VPN supports one unique SA pair per tunnel (where a pair means one inbound, and one outbound). Some policy-based devices create an SA for each ACL entry. In this situation, you must consolidate your rules and then filter unwanted traffic.
+
+To create a dynamic VPN connection, in addition to the static VPN connection requirements, the VPN Concentrator must be able to:
```

Comment:
This is probably my mistake - we use "VPN endpoint" then here switch to "VPN Concentrator". I think we should consistently use one or the other. My personal preference would be "endpoint" but both are fine.

Reply:
I prefer endpoint too.